notebook/spec2vec/tutorial.ipynb

```diff
@@ -82,8 +82,10 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
+   "execution_count": 28,
+   "metadata": {
+    "tags": []
```

Comment:
in the spec2vec notebook it misses to talk about the `metadata_list` and the `ion_mode`. honestly I think the "talk" we can literally copy from the python notebooks